src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Sql/Instrumentation.xml

```diff
@@ -47,6 +47,7 @@ SPDX-License-Identifier: Apache-2.0
       <!-- MS SQL flagship data access driver -->
```

Comment:
Do we want a comment to reflect the Core/Framework difference, like we do above?

Reply:
I'll add some comments in the instrumentation to make it clear. The separate XML elements above this are due to the fact that they are entirely separate assemblies